src/exercism/RnaTranscription.java

```diff
@@ -0,0 +1,34 @@
+class RnaTranscription {
+
+    String transcribe(String dnaStrand) {
+        String result = "";
+        for (int i = 0; i < dnaStrand.length(); i++) {
+            char c = dnaStrand.charAt(i);
+            if (c == 'G') {
```

Comment:
Please move the logic from line 7-17 to a new static function 'transformChar'

Reply:
please look at my PR, that's what am I doing